### DISCUSSION TITLE
Blog Post: Deep Dive into Cross Contract

### DISCUSSION THREAD

**Discussion by altugbakan** [stellar/sorobanathon#83] [General]
Hello everyone!

Today was a productive day for me, so I have written another deep dive technical tutorial about the Cross Contract quest this time. As usual, you can find my post on dev.to [here](https://dev.to/altug/soroban-quest-cross-contract-kfg).

I hope you guys continue to enjoy my content!

**Comment by achallagundla** [comment]
Heya @altugbakan  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1300 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.

**Comment by achallagundla** [comment]
Love your blog posts!! Would love to see you make more around general soroban and blogging your journey on you learning soroban

**Reply by altugbakan** [reply]
Thanks! Looking forward to learn more!
